### PR TITLE
Neuro branding

### PR DESCRIPTION
**Pull request by asvetlov**
Rename Neuromation -> Neuro

@mariyadavydova please check

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Name | Description|
 |_\--neuromation-config PATH_|Path to config file.|
 |_\--show-traceback_|Show python traceback on error, useful for debugging the tool.|
 |_--color \[yes &#124; no &#124; auto]_|Color mode.|
-|_\--disable-pypi-version-check_|Don't periodically check PyPI to determine whether a new version of Neuromation CLI is available for download.|
+|_\--disable-pypi-version-check_|Don't periodically check PyPI to determine whether a new version of Neuro CLI is available for download.|
```
Comment (mariyadavydova) [inline README.md]: `... of Neuro Platform CLI...`

File: docs/client_reference.rst [hunk 2]
```diff
@@ -5,15 +5,15 @@ Client class
 
 .. class:: Client
 
-   Neuromation client.
+   Neuro client.
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `Neuro Platform client`

File: docs/client_reference.rst [hunk 3]
```diff
@@ -5,15 +5,15 @@ Client class
...
 
    For creating a client instance use :class:`Factory` or :func:`get`.
 
    The class provides access to neuromation subsystems like *jobs* or *storage*.
 
    .. attribute:: username
 
-      User name used for working with Neuromation platform, read-only :class:`str`.
+      User name used for working with Neuro platform, read-only :class:`str`.
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `...with Neuro Platform...` (capital `P`)

File: docs/client_reference.rst [hunk 4]
```diff
@@ -40,12 +40,12 @@ Client class
 
    .. attribute:: parse
 
-      A set or helpers used for parsing different Neuromation API definitions, see
+      A set or helpers used for parsing different Neuro API definitions, see
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `...different Neuro Platform API...`

File: docs/client_reference.rst [hunk 5]
```diff
@@ -40,12 +40,12 @@ Client class
...
       :class:`Parser` for details.
 
    .. comethod:: close()
 
-      Close Neuromation client, all calls after closing are forbidden.
+      Close Neuro client, all calls after closing are forbidden.
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `...Neuro Platform client...`

File: docs/client_reference.rst [hunk 6]
```diff
@@ -54,9 +54,9 @@ Preset
 ======
 
    *Read-only* :class:`~dataclasses.dataclass` for describing a job configuration
-   provided by Neuromation platform.
+   provided by Neuro platform.
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `...by Neuro Platform` (capital `P`)

File: docs/client_reference.rst [hunk 7]
```diff
@@ -54,9 +54,9 @@ Preset
...
 
-   Presets list is loaded on login to the Neuromation platform and depends on used
+   Presets list is loaded on login to the Neuro platform and depends on used
```
Comment (mariyadavydova) [inline docs/client_reference.rst]: `...to Neuro Platform` (without article and capital `P`)

File: docs/functions_reference.rst [hunk 8]
```diff
@@ -71,7 +71,7 @@ Config Factory
 
 .. class:: Factory(path: Optional[Path])
 
-   A *factory* that used for making :class:`Client` instances, logging into Neuromation
+   A *factory* that used for making :class:`Client` instances, logging into Neuro
```
Comment (mariyadavydova) [inline docs/functions_reference.rst]: `Neuro Platform` (`P`)

File: docs/functions_reference.rst [hunk 9]
```diff
@@ -89,7 +89,7 @@ Config Factory
       :param aiohttp.ClientTimeout timeout: optional timeout for HTTP operations, see
                                             also :ref:`timeouts`.
 
-      :return: :class:`Client` that can be used for working with Neuromation platform.
+      :return: :class:`Client` that can be used for working with Neuro platform.
```
Comment (mariyadavydova) [inline docs/functions_reference.rst]: `Neuro Platform` (`P`)

File: docs/functions_reference.rst [hunk 10]
```diff
@@ -101,7 +101,7 @@ Config Factory
                      timeout: aiohttp.ClientTimeout = DEFAULT_TIMEOUT, \
                  ) -> None
 
-      Log into Neuromation platform using in-browser authorization method.
+      Log into Neuro platform using in-browser authorization method.
```
Comment (mariyadavydova) [inline docs/functions_reference.rst]: `Neuro Platform` (`P`)

File: docs/functions_reference.rst [hunk 11]
```diff
@@ -111,13 +111,13 @@ Config Factory
       The callback should open a browser with this URL (:func:`webbrowser.open` can be
       used).
 
-      After the call Neuromation config file is created, call :meth:`get` for making a
-      client and performing Neuromation platform operations.
+      After the call Neuro config file is created, call :meth:`get` for making a
```
Comment (mariyadavydova) [inline docs/functions_reference.rst]: `Neuro Platform config files` and `Neuro Platform operations`

**Comment by asvetlov** [comment]
@mariyadavydova please review again

**Pull request merged**